cmd/osbuild-dnf-json-tests/main.go

```diff
@@ -0,0 +1,71 @@
+// This package contains tests related to dnf-json and rpmmd package.
+package main
+
+import (
+	"fmt"
+	"github.com/osbuild/osbuild-composer/internal/rpmmd"
+	"io/ioutil"
+	"log"
+	"os"
+	"os/exec"
+	"path"
+)
+
+func main() {
+	// Tests that the package wrapping dnf-json works as expected
+	dir, err := setUpTemporaryRepository()
+	defer func(dir string) {
```

Comment:
This just a small nit - I don't think the `dir` parameter is needed, I'm fine with the function being closure.